.github/release-drafter.yml

```
@@ -0,0 +1,23 @@
+name-template: 'v$RESOLVED_VERSION'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'enhancement'
+      - 'feature'
+      - 'style'
+  - title: '🐛 Bugs Fixes'
+    labels:
+      - 'bug'
+  - title: 'Documentation'
+    labels:
+      - 'documentation'
+  - title: '🧰 Development'
```

```suggestion
  - title: '🧰 Maintenance'
```

Isn't chore not more like maintenance?